cmd/bee/cmd/deploy.go

```diff
@@ -94,6 +91,31 @@ func (c *command) initDeployCmd() error {
 				swapInitialDeposit,
 				deployGasPrice,
 			)
+			if err != nil {
+				return err
+			}
+
+			optionTrxHash := c.config.GetString(optionNameTransactionHash)
+			optionBlockHash := c.config.GetString(optionNameBlockHash)
+
+			txHash, err := node.GetTxHash(stateStore, logger, optionTrxHash)
+			if err != nil {
+				return fmt.Errorf("invalid transaction hash: %w", err)
+			}
+
+			blockHash, err := node.GetTrxNextBlock(ctx, logger, stateStore, swapBackend, transactionMonitor, txHash, optionBlockHash)
```

Comment:
Does it make sense to refactor, to do save in another place, so that it would be more visible what is happening?